src/Vlingo.Actors.Tests/ProxyGeneratorTests.cs

```diff
@@ -82,4 +99,35 @@ public interface IProxyGenericMethodWithMultipleConstraint
     {
         void Write<TSource>(string id, int state) where TSource : IProxyGenTestInterface, IProxyGenTestSecondInterface;
     }
+
+    public class MyClass
```

Comment:
Is there a case we have interfaces in classes ? Seems like a weird scenario. BTW, it would be better to give classes and interfaces more explicit names like `INestedInterface`

Reply:
It's a scenario I used from the Issue description, to replicate this. I'll rename the interfaces and classes.

Reply:
Ok, I got it now.

Reply:
I agree it's a weird scenario. I think we can still support it?